lib/theme/icons.js

```diff
@@ -1159,5 +1159,16 @@ export const icons = {
         />
       </g>
     )
+  },
+  close: {
```

Comment:
![image](https://user-images.githubusercontent.com/1152768/190393211-d01095be-f0e7-4caa-896f-a69e0f07a19e.png)
I think we already have this icon,

Reply:
Just checked. In the screenshot above, it's a `Close` component imported from theme-ui, and it's a button element. So I think we need the icon (non-button).

Reply:
but is in theme-ui it means the icon has to be there somewhere right? 
![image](https://user-images.githubusercontent.com/1152768/190419165-44ae98f2-4b21-44f9-824c-e09c90647461.png)

Reply:
Anyway, if it changes a lot of our coding style and the way to theme the component or import the icon, i'm in favor of adding the new icon instead of importing the old from theme-ui